<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>304</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>304</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IJGenerater</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img/zip.ico</normaloff>:/img/zip.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>文件信息</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="creditCode_label">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>机构代码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="creditCode_line">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>BlankCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="reportCode_label">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>报表代码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="reportCode_line">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>BlankCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="date_label">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>报表日期</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="date_line">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>BlankCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="open_button">
       <property name="font">
        <font>
         <family>Microsoft JhengHei UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>打开文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generater_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Microsoft JhengHei UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>生成IJ</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/config/config.ini</normaloff>:/config/config.ini</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>304</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="config_action"/>
   </widget>
   <widget class="QMenu" name="about_menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="donate_action"/>
    <addaction name="about_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="about_menu"/>
  </widget>
  <action name="config_action">
   <property name="text">
    <string>修改配置文件</string>
   </property>
  </action>
  <action name="donate_action">
   <property name="text">
    <string>赞赏作者</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
